ui/css/perf.css

```diff
@@ -287,6 +287,14 @@ th {
   transition: color 150ms ease-in-out;
 }
 
+.compare-table .retrigger-btn {
```

Comment:
Wouldn't this work to just assign it to`.retrigger-btn` and not `.compare-table`?